COMETwebapp/COMETwebapp/Shared/NavMenu.razor

```diff
@@ -0,0 +1,29 @@
+﻿<div class="top-row ps-3 navbar navbar-dark">
```

Comment:
Please add File header. Maybe discuss with @samatrhea what it should look like in Blazor/Razor.